README.md

```diff
@@ -19,29 +19,38 @@ by setting the `MAVEN_OPTS` environment variable to `-Xmx2g -XX:MaxPermSize=1g`.
 
 # Run
 
-To run, you'll need to install Spark. If you are just evaluating locally, you can use
+Gnocchi is built on top of [Apache Spark](http://spark.apache.org). If you are just evaluating locally, you can use
 [a prebuilt Spark distribution](http://spark.apache.org/downloads.html). If you'd like to
 use a cluster, refer to Spark's [cluster overview](http://spark.apache.org/docs/latest/cluster-overview.html).
 
 Once Spark is installed, set the environment variable `SPARK_HOME` to point to the Spark
-installation root directory. Then, you can run `gnocchi` via `./bin/gnocchi-submit`.
+installation root directory. 
 
-We include test data. You can run with the test data by running:
+The target binaries are complied to the `bin/` directory. Add them to your path with 
 
 ```
-./bin/gnocchi-submit regressPhenotypes testData/sample.vcf testData/samplePhenotypes.csv testData/associations -saveAsText
+echo "export PATH=[GNOCCHI INSTALLATION DIR]/gnocchi/bin:\$PATH" >> $HOME/.bashrc
+source $HOME/.bashrc
+```
+
+You can then run `gnocchi` via `gnocchi-submit`, or open a shell using `gnocchi-shell`.
+
+Test data is included. You can run with the test data by running:
+
+```
+gnocchi-submit regressPhenotypes testData/5snps10samples.vcf testData/10samples5Phenotypes2covars.txt ADDITIVE_LINEAR testData/associations -saveAsText -phenoName pheno1 -covar -covarFile testData/10samples5Phenotypes2covars.txt -covarNames pheno4,pheno5
```

Comment:
This is a pretty serious command to use as an example without explanation. If you're trying to point to the test data, you could just give them the path. For the users with less experience, this will be overwhelming and may not convey what you're trying to convey.